docs/client-side/using-gulp.rst

```diff
@@ -0,0 +1,219 @@
+Using Gulp
+========================================
+
+By ``_
+
+Modern web development has lots of moving parts. To build a typical web app, you might:
+
+-	Compile LESS or SASS files to CSS.
+-	Compile CoffeeScript or TypeScript files to JS.
+-	Bundle and minify your JS files.
+-	Run tools like JSHint.
+
+A *task runner* is an app that automates these routine development tasks. Visual Studio 2015 provides built-in support for two popular JavaScript-based task runners, Gulp and Grunt. 
+
+Using Gulp
+----------
+Here are the general steps to use Gulp in your ASP.NET project. This topic will cover each in more detail.
+
+1.	Use NPM to add the Gulp.js package.
+
+#.	Use NPM to add one or more `Gulp plugins <http://gulpjs.com/plugins>`_. By itself, Gulp is just an engine for running tasks. Plugins are the modules that actually do the work.
+
+#.	Define Gulp tasks.
+
+#.	Run tasks in Task Runner Explorer.
+
+#.	Bind tasks to build events.
+
+Empty vs. Web Site Projects
+---------------------------
+New ASP.NET Web Applications created in Visual Studio 2015 start with the *Empty* or *Web Site* templates. 
+ 
```

Comment:
There is also a Web API template, which also doesn't configure NPM/Gulp. Maybe phrase this as the Web Site template has step (1) done already.